editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -5461,6 +5460,13 @@ VisualShaderEditor::VisualShaderEditor() {
 	add_options.push_back(AddOption("TransformConstant", "Transform", "Variables", "VisualShaderNodeTransformConstant", TTR("Transform constant."), {}, VisualShaderNode::PORT_TYPE_TRANSFORM));
 	add_options.push_back(AddOption("TransformUniform", "Transform", "Variables", "VisualShaderNodeTransformUniform", TTR("Transform uniform."), {}, VisualShaderNode::PORT_TYPE_TRANSFORM));
 
+	// UTILITY
+
+	add_options.push_back(AddOption("DistanceFade", "Utility", "", "VisualShaderNodeDistanceFade", TTR(""), {}, VisualShaderNode::PORT_TYPE_SCALAR, TYPE_FLAGS_FRAGMENT, Shader::MODE_SPATIAL));
```

Comment:
Also, @paddy-exe - you've missed the description here - what do you suggest?

Reply:
Ah, sorry about that. I'd base it on the proximity fade node description but "... distance to the camera"